thoth-app/src/models/location/location_query.rs

```diff
@@ -0,0 +1,43 @@
+use serde::Deserialize;
```

Comment:
I can't see where this module is used.

Reply:
Used on lines 34 and 40: `#[derive(Debug, Clone, Default, Serialize, Deserialize, PartialEq, Eq)]`

Reply:
Sorry, I meant the location_query module itself.

Reply:
Yes, you're right... I think this was leftover from an earlier iteration, before everything was going through `Publication`. I've deleted it.